scripts/globals/mobskills/choke_breath.lua

```diff
@@ -0,0 +1,29 @@
+---------------------------------------------
+--  Choke Breath
+--
+--  Description: Damages enemies in a fan-shaped area of effect. Additional effect: Paralysis & Silence
+--  Type: Magical
+---------------------------------------------
+require("scripts/globals/mobskills")
+require("scripts/globals/settings")
+require("scripts/globals/status")
+---------------------------------------------
+local mobskill_object = {}
+
+mobskill_object.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskill_object.onMobWeaponSkill = function(target, mob, skill)
+    local dmgmod = 1
+    local info = xi.mobskills.mobMagicalMove(mob, target, skill, mob:getWeaponDmg() * 2, xi.magic.ele.WIND, dmgmod, xi.mobskills.magicalTpBonus.MAB_BONUS, 1)
+    local dmg = xi.mobskills.mobFinalAdjustments(info.dmg, mob, skill, target, xi.attackType.MAGICAL, xi.damageType.WIND, xi.mobskills.shadowBehavior.NUMSHADOWS_3)
+
+    xi.mobskills.mobStatusEffectMove(mob, target, xi.effect.PARALYSIS, 1, 0, 60)
```

Comment:
Ideally put in an ApplyPlayerResistance() here so we can properly calc resists unless this is always full dura.